### PR TITLE
bundle specified level endpoints with the rest of the levels

### PR DESCRIPTION
**Pull request by adamkorynta**
fix the path parameter so that the OpenAPI isn't adding both "level-id" and "specified-level-id"

### CODE REVIEW

File: cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java [hunk 1]
```diff
@@ -42,23 +38,23 @@
...
+
+import static com.codahale.metrics.MetricRegistry.name;
+import static cwms.radar.api.Controllers.*;
+import static cwms.radar.data.dao.JooqDao.getDslContext;
 
 
 public class SpecifiedLevelController implements CrudHandler {
     private static final Logger logger = Logger.getLogger(SpecifiedLevelController.class.getName());
-    private static final String TAG = "Specified Levels";
+    private static final String TAG = "Levels";
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java:57]: wait, shouldn't this one actually say Specified Levels?

Comment (MikeNeilson) [reply cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java:57]: oh wait, I see now. These are intended to be lumped under the same swagger ui box not in separate groupings.

**Review by MikeNeilson — DISMISSED** [review]

File: cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java (hunk 1 above)
Comment (adamkorynta) [reply cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java:57]: yeah, in order to keep it consistent with ratings. I was getting annoyed scrolling to the bottom of swagger for the specified level endpoints but the levels endpoints were hidden in the middle.

**Comment by adamkorynta** [comment]
rebased onto develop due to resolve conflicts

**Review by rma-rripken — APPROVED** [review]

**Pull request merged**